files/default/disable_realm.groovy

```diff
@@ -0,0 +1,8 @@
+import groovy.json.JsonSlurper
```

Comment:
Since you split the script in 2, you don't need JSON structure anymore:
```
import org.sonatype.nexus.security.realm.RealmManager;

container.lookup(RealmManager.class.getName()).disableRealm(args);
```